arangod/Aql/Query.cpp

```diff
@@ -950,7 +950,7 @@ QueryResult Query::explain() {
     }
 
     enterState(QueryExecutionState::ValueType::LOADING_COLLECTIONS);
-    _ast->validateAndOptimize(*_trx);
+    _ast->validateAndOptimize(*_trx, /*optimizeNonCacheable*/ true);
```

Comment:
I prefer to use structs instead of bools as arguments, then this would not only be easily extensible, but we could also use aggregate initialization instead of comments: `_ast->validateAndOptimize(*_trx, { .optimizeNonCacheable = true });`
But this is just a remark, no need to change to code!

Reply:
Adjusted the code according to your suggestion.